stix_shifter_modules/secretserver/stix_translation/json/event_from_stix_map.json

```diff
@@ -13,25 +13,28 @@
       ]
     }
   },
-
-  "user-account": {
+   "user-account": {
     "fields": {
-      "user_id": [
-        "UserId"
-      ]
+      "user_name": ["username"],
```

Comment:
This property needs to be `display_name`, there is no `user_name` property on the user-account object.